data/data-pipeline/data_pipeline/etl/sources/michigan_ejscreen/README.md

```diff
@@ -0,0 +1,13 @@
+# Michigan EJSCREEN
+
+The Michigan EJSCREEN description and publication can be found [here](https://deepblue.lib.umich.edu/bitstream/handle/2027.42/149105/AssessingtheStateofEnvironmentalJusticeinMichigan_344.pdf).
+
+
+#### Some notes about the input source data:
+
+1. There are two pertinent columns used - `EJ_Score_Cal_Min` and `Pct_CalMin`.
+2. The "Cal" prefix reflects California's methodology omitting racial and ethnic data and the "Min" (please see page 37 - 39 in the above reference) prefix reflects Minnesota's approach to including this data.
```

Comment:
Did you mean to write Minnesota here? If so, what does Minnesota have to do with this?

Reply:
Yes, That was intentional. Brett provided this detail in our email correspondence. Minnesota Pollution Control Agency’s (MPCA) is cited and applied when comparing their methodology (please see page 37, 82, Appendix E in the referenced white paper). A few of the input columns represent the adoption of MCPA's methodology as I understand it.

Reply:
Could you add some commentary to this effect? Otherwise I imagine future readers of this would be somewhat confused.

Reply:
I have added more details. I think this should be sufficient?